examples/apps/attributable-map/src/view.ts

```diff
@@ -0,0 +1,141 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { IRuntimeAttributor } from "@fluid-experimental/attributor";
+import { IHitCounter, greenKey, redKey, ITinyliciousUser } from "./dataObject";
+
+export function renderHitCounter(
+	hitCounter: IHitCounter,
+	runtimeAttributor: IRuntimeAttributor | undefined,
+	div: HTMLDivElement,
+) {
+	// Create wrapper div and add styling
+	const wrapperDiv = document.createElement("div");
+	wrapperDiv.style.display = "flex";
+	wrapperDiv.style.flexDirection = "column";
```

Comment:
it looks like most of our data object examples use react for the view model. any reason you didn't do that here? I'm not super opposed if comment on less dom manipulation is addressed, just curious.

Reply:
Actually I referred to the design and implementation of the example `external-views` (according to this comment in previous PR https://github.com/microsoft/FluidFramework/pull/15101#discussion_r1169013790), which does not use react for view model. I could change that to the react in the future if necessary

Reply:
nah, it's fine. I think I just sampled poorly. It's not like this demo gains much from using any particular frontend framework anyway. I think I'm happy if you reduce dom element manipulation here to just be things that are necessary (i.e. static stuff gets moved to html)